load_into_pg.py

```diff
@@ -95,13 +203,12 @@ def handleTable(table, keys, insertJson, dbname, mbDbFile, mbHost, mbPort, mbUse
                                                 for row_attribs in rows
                                             ]
                                         )
-
                             if len(valuesStr) > 0:
                                 cmd = 'INSERT INTO ' + table + \
                                       ' VALUES\n' + valuesStr + ';'
                                 cur.execute(cmd)
                                 conn.commit()
-                        six.print_('Table processing took {:.1f} seconds'.format(time.time() - start_time))
+                        six.print_('Table processing took {1:.1f} seconds'.format(table, time.time() - start_time))
```

Comment:
Why not 'Processing table {} took ...' ?

Reply:
I might have had the idea to log it. Well spotted!
I added it.